modules/core/02-client/keeper/keeper_test.go

```diff
@@ -642,3 +643,59 @@ func (suite *KeeperTestSuite) TestIBCSoftwareUpgrade() {
 		})
 	}
 }
+
+func (suite *KeeperTestSuite) TestIBCScheduledUpgradeProposal() {
```

Comment:
This can also be deleted I think once https://github.com/cosmos/cosmos-sdk/issues/22779 is solved, but its also no harm keeping around as it routes a msg for scheduleIBCUpgrade through gov

Reply:
Maybe a comment with a link to that for context?